.CI/create_feedstocks

```diff
@@ -7,21 +7,32 @@ set -e
 # recipes from. Currently this is `master`.
 git checkout "${TRAVIS_BRANCH}"
 
-# Download and setup Miniconda
-mkdir -p ~/miniconda_staging
-pushd ~/miniconda_staging
-curl -L -O https://raw.githubusercontent.com/conda-forge/conda-smithy/e976c7e84bb3c4846e7562afd1a42c4b535b51f5/bootstrap-obvious-ci-and-miniconda.py
-python bootstrap-obvious-ci-and-miniconda.py ~/miniconda x64 3 --without-obvci && source ~/miniconda/bin/activate root
-popd
-rm -rf ~/miniconda_staging
+# Install Miniconda.
+echo ""
+echo "Installing a fresh version of Miniconda."
+[ ! -f ~/cache/miniconda.sh ] && curl -L https://repo.continuum.io/miniconda/Miniconda3-latest-MacOSX-x86_64.sh > ~/cache/miniconda.sh
+bash ~/cache/miniconda.sh -b -p ~/miniconda
+source ~/miniconda/bin/activate root
 
+# Reuse packages from the cache.
+mkdir -p ~/cache/pkgs
+rm -rf ~/miniconda/pkgs
+mv ~/cache/pkgs ~/miniconda/pkgs
```

Comment:
Yeah, that could work. We could also just start caching everything like we did [here]( https://github.com/conda-forge/conda-forge.github.io/blob/84f85c4eaa2798b43389d223eac82e3536ea4c75/.travis.yml#L23-L24 ).

After playing with caching a bit, I think we'd be better off to just cache all packages downloaded. PRs have separate caches from `master` and the repo as a whole anyways. So we don't need to worry about getting a massive cache of useless things. Plus it should make it easier for people to iterate on a PR.